src/components/Modals/NewCourse/CourseConflictModal.vue

```diff
@@ -104,17 +148,55 @@ export default defineComponent({
       let unresolved = false;
       let unselected = false;
       for (let i = 0; i < this.selectedReqsPerConflict.length; i += 1) {
-        if (this.selectedReqsPerConflict[i].length > 1) {
+        const conflict = this.selectedReqsPerConflict[i];
+        const numReqsSelected = this.countNumberReqsSelected(conflict);
+        if (numReqsSelected > 1) {
           unresolved = true;
         }
 
-        if (this.selectedReqsPerConflict[i].length === 0) {
+        if (numReqsSelected === 0) {
           unselected = true;
         }
       }
       this.hasUnresolvedConflicts = unresolved;
       this.hasUnselectedConflicts = unselected;
     },
+    // count number of reqs selected for a conflict (i.e. set to true in the map)
+    countNumberReqsSelected(conflict: Map<string, boolean>) {
+      let count = 0;
+      conflict.forEach((value: boolean) => {
+        if (value) count += 1;
+      });
+      return count;
+    },
+    // only show the selectable req warning under the first req group, and only if there are selectable reqs
+    shouldShowSelectableWarning(index: number): boolean {
+      const maxNumSelfChecks = Math.max(...this.numSelfChecksPerConflict);
+      return index === 1 && maxNumSelfChecks > 0;
+    },
+    // get the reqs in conflictReqIds that are in conflict, based on course with uniqueID
+    // self check requirements not in conflict with the other reqs will be excluded
+    getReqsInConflict(uniqueID: string | number, conflictReqIds: string[]): string[] {
+      const constraintViolations = getConstraintViolationsForSingleCourse(
+        { uniqueId: uniqueID },
+        conflictReqIds,
+        (reqA, reqB) =>
+          allowCourseDoubleCountingBetweenRequirements(
+            store.state.userRequirementsMap[reqA],
+            store.state.userRequirementsMap[reqB]
+          )
+      );
+
+      const validConflicts = constraintViolations.courseToRequirementsInConstraintViolations.get(
+        this.selectedCourse.uniqueID
+      );
+      let firstConflict: string[] = [];
+      if (validConflicts) {
+        [firstConflict] = [...validConflicts];
+      }
+
+      return firstConflict;
+    },
```

Comment:
We should check the selectable requirements one by one and see if the conflicts are disjoint (size 1 vs size 2). If so, append the selectable requirement. (instead of relying on the order of conflicts)